src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/service/JiraBuildInfoSenderImpl.java

```diff
@@ -92,15 +81,15 @@ public JiraBuildInfoSenderImpl(
      * @param request - JiraBuildInfoRequest::site is ignored and jiraSite is used instead
      */
     public JiraSendInfoResponse sendBuildInfoToJiraSite(
-            @Nonnull final JiraCloudSiteConfig siteConfig,
+            @Nonnull final JiraCloudSiteConfig2 siteConfig,
             @Nonnull final JiraBuildInfoRequest request) {
 
-        final String resolvedSiteConfig = siteConfig.getSite();
+        final String jiraSite = siteConfig.getSite();
```

Comment:
Just renamed `resolvedSiteConfig` to `jiraSite`, because that makes it better understandable.